design/ui/pagination/README.mdx

```diff
@@ -0,0 +1,170 @@
+---
+name: Pagination
+---
```

Comment:
We will eventually have much more in the header schema of these documents... including the component name. I think we can make a lot of the MDX generation automatic from that with a template. Things like importing the component based on the name, adding it automatically to the Props component, etc.